test/e2e/json-rpc/switchEthereumChain.spec.js

```diff
@@ -158,7 +158,36 @@ describe('Switch Ethereum Chain for two dapps', function () {
         });
 
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
+        // Switch to Dapp One and connect it
+        await driver.switchToWindowWithUrl(DAPP_URL);
+        await driver.findClickableElement({
```

Comment:
What is the purpose of finding the element prior to clicking it?

Reply:
It's a common pattern we have used in the spec files, should we remove it?

Reply:
Yes, this pattern does not serve any purpose. The steps within `findClickableElement` are also within `clickElement`, making this totally redundant.

Reply:
Please address this in a follow up @NidhiKJha